### ISSUE TITLE
Mention environment variables in a Windows-friendly way

### ISSUE THREAD

**Issue by mikhailshilkov** [pulumi/docs#2057]
The [PaC docs](https://www.pulumi.com/docs/get-started/policy-as-code/authoring-a-policy-pack/) mention setting env variables from the command itself:

> PULUMI_DEBUG_COMMANDS=true pulumi policy new aws-typescript

This won't work on Windows. I think we should suggest setting this variable for the session at the beginning of the tutorial. Env variable syntax also varies between PowerShell and cmd, so we need to figure the best wording and then apply it throughout.

**Comment by justinvp** [comment]
I made some changes for the PaC docs in https://github.com/pulumi/docs/pull/2056 to mention how to set the environment variable for Windows (CMD and PowerShell). Though, I didn't do it consistently. For macOS/Linux, I show just setting before running the command, and for Windows, I set it for the session before each command (which becomes redundant with subsequent commands later in the guides). Agree it'd be better to just mention setting it for the session at the outset for all operating systems. My only worry is if people are skimming and miss that step, then the commands later on the page aren't going to work. We may need to have a small note after each command to mention the env variable needs to be set with an anchor link to the top of the page where it's explained.
